.prettierrc.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
I'd prefer we use Prettier without any settings as we offload any decision making to that team (see version Prettier 2 changes) and I don't believe there's an easy way to share a Prettier config across repos, which means we have to keep them in sync manually.

Reply:
> I don't believe there's an easy way to share a Prettier config across repos

Scratch that: https://prettier.io/docs/en/configuration.html#sharing-configurations

If the general consensus is to define our own settings then let's pull them out into `@stylelint/prettier-config` and roll them out across the org. We can do that in a follow-up pull request, one that probably uses [`remark-preset-lint-stylelint`](https://github.com/stylelint/remark-preset-lint-stylelint/pulls) too.

Reply:
Pull request pending at https://github.com/stylelint/prettier-config/pull/1